src/portuguese.js

```diff
@@ -0,0 +1,43 @@
+
+class PresenteIndicativo {
+  conjugate (word, info) {
+    let ending = [];
+    switch (word.slice(-2)) {
+      case 'ar':
+        ending = info.singular ? ['o', 'as', 'a'] : ending = ['amos', 'ais', 'am'];
+        break;
+      case 'er':
+        ending = info.singular ? ['o', 'es', 'e'] : ['emos', 'eis', 'em'];
```

Comment:
Yes, you might be right, I thought about it, but I was following the conjugators already implemented and none of them uses 'subject', the French conjugator uses 'person', and 'singular', so I thought that was something you all had agreed to keep. At least the options for the conjugate method would be consistent.